tests/apps/asgi/commands/files/dummy_server.py

```diff
@@ -40,6 +40,7 @@ async def receive():
         "method": "GET",
         "query_string": b"",
         "headers": [],
+        "_body": b"null",
```

Comment:
This is great. I tried that and it works great. I just need to add `httpx` to `test-requirements.txt`and replace 
```python
async def dummy_server(app: t.Union[str, t.Callable] = "app:app"): 
```
with 
```python
async def dummy_server(app: t.Union[t.Any, t.Callable] = "app:app"): 
```
to keep mypy happy.

Reply:
Oh yeah, I see the mypy error now.

You can also do this:

```python
app = t.cast(t.Callable, getattr(module, app_name))
```

Instead of setting the arg type to `Any`.